controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -265,32 +259,17 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageRequest(
 		return err
 	}
 
-	cephClientName = generateHashForCephClient(cephFsStorageRequestName, "provisioner")
-	if err := r.reconcilePre4_19CephClient(
-		ctx,
-		cephClientName,
-		storageConsumer.Namespace,
-		storageConsumer,
-	); err != nil {
-		return err
-	}
-	cephClientName = generateHashForCephClient(cephFsStorageRequestName, "node")
-	if err := r.reconcilePre4_19CephClient(
-		ctx,
-		cephClientName,
-		storageConsumer.Namespace,
-		storageConsumer,
-	); err != nil {
-		return err
-	}
-
 	cephFsStorageRequest := &metav1.PartialObjectMetadata{}
 	cephFsStorageRequest.SetGroupVersionKind(ocsv1alpha1.GroupVersion.WithKind("StorageRequest"))
 	cephFsStorageRequest.Name = cephFsStorageRequestName
 	cephFsStorageRequest.Namespace = storageConsumer.Namespace
 	if err := r.Client.Get(ctx, client.ObjectKeyFromObject(cephFsStorageRequest), cephFsStorageRequest); client.IgnoreNotFound(err) != nil {
 		return err
 	} else if cephFsStorageRequest.UID != "" {
+		if err := r.reconcilePre4_19CephClients(ctx, cephFsStorageRequest.UID, storageConsumer); err != nil {
+			return err
+		}
```

Comment:
Why did we move the code around? wouldn't this call make more sense in the same position of the original code?